docs/content/middlewares/forwardauth.md

```diff
@@ -266,7 +266,12 @@ http:
 
 #### `tls.caOptional`
 
-TODO add description.
+The policy followed by the server for TLS Client Authentication.
+Only used if `tls.ca` is defined.
+
+- `true`: VerifyClientCertIfGiven
+- `false`: RequireAndVerifyClientCert
```

Comment:
Do we have a reference to what `RequireAndVerifyClientCert ` is?

Reply:
nope